documentation/how-to/attributes-form.en.md

```diff
@@ -23,7 +23,7 @@ The field widgets from QGIS are supported on a best effort basis and optimized f
 | Value Relation     | :material-check: |                                                                                                                                                                                                        |
 | UUID Generator     | :material-check: |                                                                                                                                                                                                        |
 | QML / HTML Widget  | :material-check: |                                                                                                                                                                                                        |
-| Others             | :material-close: | [Funding](../../get-started/support.md#development-and-custom-apps)                                                                                                                                      |
+| Others             | :material-close: | [Funding](././get-started/support.md#development-and-custom-apps)                                                                                                                                      |
```

Comment:
```suggestion
| Others             | :material-close: | [Funding](./get-started/support.md#development-and-custom-apps)                                                                                                                                      |
```

@lucienicolier `./` means "the current directory", repeating it doesn't have any influence (maybe we could even skip it completely here)